operations/services.md

```diff
@@ -272,6 +272,188 @@ Returns all products offered together with the specified services.
+                "Discriminator": "Product",
+                "Value": {
+                    "ProductId": "122fd92d-c561-4995-8ebc-ac1c00d1eaa8",
+                    "ActionType": "Add"
+                }
+            }
+        }
+    ],
+    "Rates": [
+        {
+            "Id": "8a13f4e8-0274-4bcc-b2d4-ac1c00d1cd3f",
+            "GroupId": "e4a9d8d1-5793-4d35-954e-ac1c00d1eaa8",
+            "ServiceId": "ea80bbca-372f-4550-8e48-ac1c00d1cd20",
+            "BaseRateId": null,
+            "IsActive": true,
+            "IsPublic": true,
+            "Name": "Fully Flexible",
+            "ShortName": "FF",
+            "ExternalNames": {}
+        }
+    ],
+    "RateGroups": [],
+    "ResourceCategories": [],
+    "BusinessSegments": []
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Rules` | array of [Rule](#rule) | required | Rules applied with reservations. |
+| `RuleActions` | array of [Rule action](#rule-action) | required | Rule actions applied in rules. |
+| `Rates` | array of [Rate](#rate) | required | Rates used for conditions. |
+| `RateGroups` | array of [Rate group](#rate-group) | required | Rate groups used for conditions. |
+| `ResourceCategories` | array of [Resource category](enterprises.md#resource-category) | required | Resource categories used for conditions. |
+| `BusinessSegments` | array of [Business segment](#business-segment) | required | Business segments used for conditions. |
+
+#### Rule
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Id` | string | required | Unique identifier of the rule. |
+| `Conditions` | [Rule conditions](#rule-conditions) | required | Conditions of the rule. |
+
+#### Rule conditions
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `RateId` | [Rule condition](#rule-condition) | required | Condition based on [Rate](#rate). |
+| `RateGroupId` | [Rule condition](#rule-condition) | required | Condition based on [Rate group](#rate-group). |
+| `BusinessSegmentId` | [Rule condition](#rule-condition) | required | Condition based on [Business segment](#business-segment). |
+| `ResourceCategoryId` | [Rule condition](#rule-condition) | required | Condition based on [Resource category](enterprises.md#resource-category). |
+| `ResourceCategoryType` | [Rule condition](#rule-condition) | required | Condition based on [Resource category type](enterprises.md#resource-category-type). |
+| `Origin` | [Rule condition](#rule-condition) | required | Condition based on [Service order origin](#service-order-origin). |
+| `MinimumTimeUnitCount` | string | required | Condition based on minimum amount of time units. |
+| `MaximumTimeUnitCount` | string | required | Condition based on maximum amount of time units. |
+
+### Rule condition
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Value` | string | required | Unique identifier of condition or [Service order origin](#service-order-origin) of the reservation. |
```

Comment:
I'd change the description to something like "Value of the condition depending on the property. E.g. service order origin in case of origin condition or unique identifier of a rate in case of rate condition."